lib/jobs/idListProcessor.js

```diff
@@ -0,0 +1,6 @@
+const scrapeIds = require('./scrapeIds');
```

Comment:
i don't think you need this file. `scrapeIds` is already the processor you want. this adds no functionality